<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>951</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: none;
outline: none;
}
QPushButton:foucs{
background-color: rgb(240,240,240);
}
QPushButton:hover{
background-color: rgb(240,240,240);
}</string>
    </property>
    <property name="text">
     <string>Function 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: none;
outline: none;
}
QPushButton:foucs{
background-color: rgb(240,240,240);
}
QPushButton:hover{
background-color: rgb(240,240,240);
};
background-color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Function 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: none;
outline: none;
}
QPushButton:foucs{
background-color: rgb(240,240,240);
}
QPushButton:hover{
background-color: rgb(240,240,240);
};
background-color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Function 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closebutton">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>10</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Icons/delete.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minimizebutton">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>10</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Icons/minimize.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Username">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Icons/user.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="page4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: none;
outline: none;
}
QPushButton:foucs{
background-color: rgb(240,240,240);
}
QPushButton:hover{
background-color: rgb(240,240,240);
}</string>
    </property>
    <property name="text">
     <string>Function 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(1, 62, 122);
border: none;
outline: none;</string>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>3</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Icons/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="page5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: none;
outline: none;
}
QPushButton:foucs{
background-color: rgb(240,240,240);
}
QPushButton:hover{
background-color: rgb(240,240,240);
}</string>
    </property>
    <property name="text">
     <string>Function 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: none;
outline: none;
}
QPushButton:foucs{
background-color: rgb(240,240,240);
}
QPushButton:hover{
background-color: rgb(240,240,240);
}</string>
    </property>
    <property name="text">
     <string>Function 6</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
